include/starpu_cublasLt.h

```diff
@@ -0,0 +1,61 @@
+/* StarPU --- Runtime system for heterogeneous multicore architectures.
+*
+ * Copyright (C) 2010-2024  University of Bordeaux, CNRS (LaBRI UMR 5800), Inria
+ *
+ * StarPU is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or (at
+ * your option) any later version.
+ *
+ * StarPU is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU Lesser General Public License in COPYING.LGPL for more details.
+ */
+
+#ifndef __STARPU_CUBLASLT_H__
+#define __STARPU_CUBLASLT_H__
+
+#ifdef STARPU_USE_CUDA
+#include <cublasLt.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    /**
+       @ingroup API_CUDA_Extensions
+       @{
+    */
```

Comment:
Please do not add these four spaces, the `extern "C" {` is really expected in the StarPU code, so indenting its content is really not useful

Reply:
Fixed, my IDE was making mess